api/python/cellxgene_census/pyproject.toml

```diff
@@ -31,7 +31,7 @@ dependencies= [
     # NOTE: the tiledbsoma version must be >= to the version used in the Census builder, to
     # ensure that the assets are readable (tiledbsoma supports backward compatible reading).
     # Make sure this version does not fall behind the builder's tiledbsoma version.
-    "tiledbsoma~=1.7.0",
+    "tiledbsoma~=1.8.0",
```

Comment:
I believe tiledbsoma==1.8.0 supports python 3.12.  Do you want to support 3.12 in this release as well, or does that require additional effort?

Reply:
Looks like there are some issues with Python 3.12 in tiledbsoma so the support is marked as provisional: https://github.com/single-cell-data/TileDB-SOMA/issues/1849.

So I think it's safe to leave it out from the Census for now. I'll ask John to let us know when it's fully supported.

Reply:
sounds good. Also cross-referencing #839